src/content/self-hosted/eks/config.yaml

```diff
@@ -1,12 +1,3 @@
 license:
 
 subdomain: 
-
-cluster:
-  endpoint:
-
-ingress-nginx:
```

Comment:
Why are we removing this part? I don't get how this is related to the cluster endpoint

Reply:
We shouldn't have this snippet here, the guide isn't using it. At some point we should default to NLB, but there is an open issue about it